crud/common/collations.lua

```diff
@@ -0,0 +1,27 @@
+local json = require('json')
+
+local COLLATION_NAME_FN = 2
+
+local collations = {}
+
+collations.NONE = 'none'
```

Comment:
https://www.tarantool.io/en/doc/2.5/book/box/data_model/#collations

```
By default, when Tarantool compares strings, it uses what we call a “binary” collation.
```

Is there any difference between 'none' and 'binary' collation?
Should we add some tests for it?

Reply:
Seems it's the same
```
tarantool> box.space._collation:select()
---
- - [0, 'none', 1, 'BINARY', '', {}]
  - [1, 'unicode', 1, 'ICU', '', {'strength': 'tertiary'}]
  - [2, 'unicode_ci', 1, 'ICU', '', {'strength': 'primary'}]
  - [3, 'binary', 1, 'BINARY', '', {}]
```

Reply:
I just wanted to decrease number of `collation ~= nil and collation ~= collations.NONE` comparisons.